packages/sdk/webpack.config.js

```diff
@@ -3,13 +3,20 @@ const webpack = require("webpack")
 
 module.exports = {
 	entry: "./build/index.js",
+	experiments: {
+		outputModule: true,
+	},
 	output: {
 		path: path.resolve(__dirname, "umd"),
 		filename: "rarible-sdk.js",
 		library: {
-			name: "raribleSdk",
 			type: "umd",
 		},
+		path: path.resolve(__dirname, "esm"),
+		filename: "rarible-sdk.js",
+		library: {
+			type: "module",
+		},
```

Comment:
wrong code: it duplicates code above

Reply:
Yes. Having umd and esm builds is common practice for libraries. 

Umd is compatible with everything. Esm is not, but umd doesn't support tree shaking so adding an esm build is necessary. 

Check out this article for more info: [link](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm)